test-ca/build-a-pki.sh

```diff
@@ -174,3 +201,7 @@ for kt in rsa ecdsa eddsa ; do
 
   openssl asn1parse -in $kt/ca.cert -out $kt/ca.der > /dev/null
 done
+
+# Tidy up openssl CA state.
+rm index.txt* || true
+rm crlnumber* || true
```

Comment:
using wildcards here because openssl ca writes a variety of files, e.g. crlnumber.old, index.txt.attr, etc.